easyaudit/signals/auth_signals.py

```diff
@@ -32,9 +40,11 @@ def user_login_failed(sender, credentials, **kwargs):
         with transaction.atomic():
             request = get_current_request() # request argument not available in django < 1.11
             user_model = get_user_model()
-            login_event = LoginEvent.objects.create(login_type=LoginEvent.FAILED,
-                                                    username=credentials[user_model.USERNAME_FIELD],
-                                                    remote_ip=request.META[REMOTE_ADDR_HEADER])
+            login_event = audit_logger.login({
+                'login_type': LoginEvent.FAILED,
+                'username': getattr(user, user.USERNAME_FIELD),
```

Comment:
sorry just saw: this needs to be based on `user_model` like before. `user` doesn't exist.